README.md

```diff
@@ -171,6 +171,15 @@ $.Error :: Type
 Type comprising every Error value, including values of more specific
 constructors such as [`SyntaxError`][2] and [`TypeError`][3].
 
+#### `FiniteNumber`
+
+```haskell
+$.FiniteNumber :: Type
+```
+
+Type comprising every [`ValidNumber`](#validnumber) value except `Infinity`, 
+`-Infinity` and their object counterparts. 
```

Comment:
I would prefer to parenthesize the bit about Number objects, as it's a distraction we're only mentioning for the sake of accuracy. Essentially, I'd like people to ignore the parenthesized fragment when reading the description.

> Type comprising every [`ValidNumber`](#validnumber) value except `Infinity` and `-Infinity` (and their object counterparts).